read_zdf_vis_3d.py

```diff
@@ -1,38 +1,32 @@
-"""
-Import ZDF point cloud and visualize it.
-"""
+"""Import ZDF point cloud and visualize it."""
 
+import math
 import numpy as np
 import matplotlib.pyplot as plt
-from vtk_visualizer import plotxyzrgb
 import zivid
+import pptk
 
 
-def _main():
-
-    app = zivid.Application()
+def _display_rgb(rgb):
+    """Create and display depthmap.
 
-    # The Zivid3D.zdf file has to be in the same folder as this sample script.
-    filename_zdf = "Zivid3D.zdf"
+    Args:
+        rgb: RGB matrix
 
-    print(f"Reading {filename_zdf} point cloud")
-    frame = zivid.Frame(filename_zdf)
-
-    # Getting the point cloud
-    point_cloud = frame.get_point_cloud()
-    xyz = np.dstack([point_cloud["x"], point_cloud["y"], point_cloud["z"]])
-    rgb = np.dstack([point_cloud["r"], point_cloud["g"], point_cloud["b"]])
-
-    # Flattening the point cloud
-    flattened_point_cloud = point_cloud.reshape(-1, 6)
-
-    # Displaying the RGB image
+    """
     plt.figure()
     plt.imshow(rgb)
     plt.title("RGB image")
-    plt.show()
+    plt.show(block=False)
+
+
+def _display_depthmap(xyz):
+    """Create and display depthmap.
+
+    Args:
+        xyz: X,Y and Z co-ordinates
```

Comment:
I would call this X, Y and Z Images. Co-ordinates might seem misleading - meaning three coordinates rather than 3 x 2.stg million of them.

Reply:
I understand your comment, but I would be confused by "X, Y and Z Images". What is an "X Image"?
Maybe we could say "Set of X, Y and Z co-ordinates" or "Collection of X, Y and Z co-ordinates"

Reply:
I understand your point that the terminology I suggest is not straightforward to understand for someone not familiar with it. This is the terminology we use internally and I think it makes sense to try to make customers adopt it. To me, it sounds simple and it is a nice way to split the point cloud in 3 parts, but I agree that it needs to be clarified.

How about we use both in the samples, stg like: X, Y and Z images (collections of coordinates)

Reply:
But maybe I am wrong and just used to it, I am not sure :)

Reply:
X,Y and Z images (point cloud co-ordinates)